### PR TITLE
Feature: compress/decompress

### PR DESCRIPTION
**Pull request by neliudochka**
- Implemented CompressString class.
It has 2 methods: compress (string -> byte array), and decompress (byte array -> string)

- Added tests for CompressString class.
Used JUnit to write them.

### CODE REVIEW

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.example.steganography.algorithms
+
+import java.io.ByteArrayOutputStream
+import java.util.zip.GZIPInputStream
+import java.util.zip.GZIPOutputStream
+
+class CompressString {
+    fun compress(input: String) :ByteArray {
+        val bos = ByteArrayOutputStream()
```
Comment (AKushch1337) [inline app/src/main/java/com/example/steganography/algorithms/CompressString.kt:9]: Use linter, as `:ByteArray` looks odd

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        GZIPOutputStream(bos).bufferedWriter(Charsets.UTF_8).use { it.write(input) }
+        return bos.toByteArray()
+    }
+
+    fun decompress(input :ByteArray) :String {
+        return GZIPInputStream(input.inputStream())
```
Comment (AKushch1337) [inline app/src/main/java/com/example/steganography/algorithms/CompressString.kt:15]: Same here, use linter, please!

**Review by AKushch1337 — CHANGES_REQUESTED** [review]
- Using linter is a good practice, since you are working in Android Studio you can use the hotkey Ctrl+Alt+L. 
- Other than that all good. Well done!

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt (hunk 1 above)
Comment (neliudochka) [reply app/src/main/java/com/example/steganography/algorithms/CompressString.kt:9]: Screenshot from 2023-06-08 02-13-09

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt (hunk 2 above)
Comment (neliudochka) [reply app/src/main/java/com/example/steganography/algorithms/CompressString.kt:15]: Screenshot from 2023-06-08 02-13-09

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt (hunk 1 above)
Comment (neliudochka) [reply app/src/main/java/com/example/steganography/algorithms/CompressString.kt:9]: ok

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt (hunk 2 above)
Comment (neliudochka) [reply app/src/main/java/com/example/steganography/algorithms/CompressString.kt:15]: ok

Comment (AKushch1337) [reply app/src/main/java/com/example/steganography/algorithms/CompressString.kt:15]: 🐽

File: app/src/main/java/com/example/steganography/algorithms/CompressString.kt (hunk 1 above)
Comment (AKushch1337) [reply app/src/main/java/com/example/steganography/algorithms/CompressString.kt:9]: 🐽

**Review by AKushch1337 — APPROVED** [review]
Nice!

**Pull request merged**